services/director-v2/src/simcore_service_director_v2/core/errors.py

```diff
@@ -135,3 +136,32 @@ def __init__(self, msg: Optional[str] = None):
         super().__init__(
             msg or "Invalid configuration of the director-v2 application. Please check."
         )
+
+
+class ClusterNotFoundError(PydanticErrorMixin, SchedulerError):
+    code = "cluster not found error"
```

Comment:
This ``code`` is intended as a key and not a message, it identify the error; while the ``msg_template`` is used as a formatted message (typically denoted as ``reason`` in the error class params)


SEE e.g. https://github.com/samuelcolvin/pydantic/blob/master/pydantic/errors.py#L149